lambdas/cfn-sc-actions-provider/template.yaml

```diff
@@ -0,0 +1,101 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+Description: Service Catalog actions provider
+Resources:
+  CreateFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: python3.6
+      CodeUri: sc-actions-provider/
+      Handler: app.lambda_handler
+      Role: !GetAtt FunctionRole.Arn
+      Timeout: 30
+  AssociateFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Runtime: python3.6
+      CodeUri: sc-actions-provider/
+      Handler: associate.lambda_handler
+      Role: !GetAtt FunctionRole.Arn
+      Timeout: 30
+  FunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      ManagedPolicyArns:
+        - !Ref LogsPolicy
+        - "arn:aws:iam::aws:policy/AWSServiceCatalogAdminFullAccess"
+#        - !Ref ProviderPolicy
+  LogsPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Action:
+              - logs:CreateLogGroup
+              - logs:CreateLogStream
+              - logs:PutLogEvents
+            Resource: "*"
+
+  ProviderPolicy:
```

Comment:
not used right now, this was an attempt to define  a least privileged policy for this lambda however this policy does not work.